examples/with-mobx/components/Page.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Link from 'next/link'
+import { inject, observer } from 'mobx-react'
+import Clock from '../components/Clock'
```

Comment:
Yes my bad, also should I remove all the React import ? As I understand it is not required in next.